pandas/_libs/tslib.pyx

```diff
@@ -104,7 +110,7 @@ def ints_to_pydatetime(const int64_t[:] arr, object tz=None, object freq=None,
         str typ
         int64_t value, delta, local_value
         ndarray[object] result = np.empty(n, dtype=object)
-        object (*func_create)(int64_t, npy_datetimestruct, object, object)
+        object (*func_create)(int64_t, npy_datetimestruct, object, object, bint)
```

Comment:
Because we adjust `create_timestamps_from_ts`, need to adjust all the other `func_create` candidates to match the new signature.